pkgs/by-name/re/rerun/package.nix

```diff
@@ -22,19 +21,15 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "rerun";
-  version = "0.18.2";
+  version = "0.20.1";
   src = fetchFromGitHub {
     owner = "rerun-io";
     repo = "rerun";
     rev = version;
-    sha256 = "sha256-mQjjgRKNFSts34Lphfje9H1BLY9nybCrJ2V09nMzVDM=";
+    hash = "sha256-gDET+WtFzhPHpgRJeylfQ2W7qh8k7rf9U30jwEWmPJc=";
   };
 
-  cargoHash = "sha256-ZyjRe4M6RabSKhKCLa1ed1fsF6dkUt2a1c8C/1E48+M=";
-  # the crate uses an old rust version (currently 1.76)
-  # nixpkgs only works with the latest rust (currently 1.80)
-  # so we patch this
-  cargoPatches = [ ./rust-version.patch ];
+  cargoHash = "sha256-CIPjJSoHUAF8tCU4NNclsIViu9hWwmsVYpjvySRqqfc=";
```

Comment:
`if hostPlatform.isDarwin then "sha256-CIPjJSoHUAF8tCU4NNclsIViu9hWwmsVYpjvySRqqfc=" else "sha256-APo3AjKM3n/lS5QZg0VEW83RAZIm6T5/itADPQ4qSNs="`

Reply:
This is probably a case‐sensitivity issue that we should patch out instead.

Reply:
Note how the [new error](https://github.com/NixOS/nixpkgs/pull/358093/checks?check_run_id=33596255691) on `*-linux` mirrors the [old](https://github.com/NixOS/nixpkgs/pull/358093#issuecomment-2496093294) on `*-darwin`:

```
         specified: sha256-CIPjJSoHUAF8tCU4NNclsIViu9hWwmsVYpjvySRqqfc=
            got:    sha256-APo3AjKM3n/lS5QZg0VEW83RAZIm6T5/itADPQ4qSNs=
```

Reply:
Yeah, but that’s probably just because macOS uses a case‐insensitive filesystem. Usually we do `postFetch` stuff to fix the filenames that differ only in case.

And it’s probably a good idea to try using `useFetchCargoVendor` just in case that helps.

Reply:
Not sure if ofborg is happy enough to show that the upgrade from 0.20.1 to 0.20.2 fixed the issue with different hashes on linux / darwin. Can someone double check ?
Otherwise I'll give that "new fetcher" a try.

Reply:
Oh, wait, I already added `useFetchCargoVendor = true;`